pkg/loop/internal/pb/relayer.pb.go

```diff
@@ -2146,6 +2146,237 @@ func (x *FromAccountReply) GetAccount() string {
 	return ""
 }
 
+type VersionedBytes struct {
```

Comment:
How is this generated in relayer, it was moved to the chain_reader.proto.

Reply:
You should add it to the generate.go and `make generate` to fix it up

Reply:
Out of scope, but we should add a version of this check to enforce tidy and generate: https://github.com/smartcontractkit/chainlink/blob/develop/.github/workflows/ci-core.yml#L227-L263

Reply:
My bad, didn't commit everything properly, fixed now